pkg/debug/Dockerfile

```diff
@@ -35,18 +37,9 @@ WORKDIR /lshw-B.${LSHW_VERSION}
 RUN for patch in fix-musl-sc_long_bit.patch wrapper-for-basename.patch 15565229509455527de9ce7cbb9530e2b31d043b.patch\
  2b1c730b493d647bbab4854713571458e82a81e7.patch; do patch -p1 < $patch; done &&\
  make -C src RPM_OPT_FLAGS=-DNONLS static &&\
- cp src/lshw-static /lshw && strip /lshw
+ cp src/lshw-static /out/usr/bin/lshw && strip /out/usr/bin/lshw
 
 FROM linuxkit/sshd:666b4a1a323140aa1f332826164afba506abf597
```

Comment:
Probably, we can also move this one to scratch?

Reply:
Would you you like to help with that (as a separate PR?)?

Reply:
Here you are https://github.com/lf-edge/eve/pull/1987